pkg/secrets/config/config.go

```diff
@@ -254,21 +276,26 @@ func NewConfig(settings map[string]string) *Config {
 	retryIntervalSec := parseIntFromStringOrDefault(retryIntervalSecStr, DefaultRetryIntervalSec, MinRetryValue)
 
 	refreshIntervalStr := settings[SecretsRefreshIntervalKey]
-	refreshEnableStr   := settings[SecretsRefreshEnabledKey]
+	refreshEnableStr := settings[SecretsRefreshEnabledKey]
 
 	if refreshIntervalStr == "" && refreshEnableStr == "true" {
 		refreshIntervalStr = DefaultRefreshIntervalStr
 	}
 	// ignore errors here, if the interval string is null, zero is returned
 	refreshInterval, _ := time.ParseDuration(refreshIntervalStr)
 
+	// TODO: Should we only enable this when rotation is enabled?
```

Comment:
I'm wondering if we need this TODO. Seems that the checks for 403/404 errors (and subsequent sanitizing) would only occur if the refresh periodic is in effect, and that's only going to happen when rotation is enabled?

Reply:
Hah, I actually removed this but forgot to commit it :)